pytest_fixtures/component/os.py

```diff
@@ -2,9 +2,11 @@
 import pytest
 from nailgun import entities
 
+from robottelo.config import settings
```

Comment:
you can move this import into the fixture that needs it

Reply:
I can but when there will be another fixture that needs settings this will require another change. Is there any reason to move this import to function if it is not conflicting with anything else? I would rather keep imports in one place

Reply:
Thanks for the explanation, moving into the fixture